src/components/FeedbackSnackbar.js

```diff
@@ -15,17 +15,22 @@ const FeedbackSnackbar = (props) => {
     clearSnackbar();
   };
 
-  return (
-    <Snackbar
-      open={snackbar.isOpen}
-      autoHideDuration={6000}
-      onClose={handleClose}
-    >
-      <Alert onClose={handleClose} severity={snackbar.type}>
-        {snackbar.message}
-      </Alert>
-    </Snackbar>
-  );
+  if (snackbar.isOpen) {
```

Comment:
Added a guard because without it, if we don't redirect, a console error appears due to our impost of `""` into Alert's `severity` through `snackbar.type` from a closed Snackbar in redux.